ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```diff
@@ -2234,7 +2234,11 @@ interface Port-Channel137
    description Traffic Engineering Interface
    no switchport
    ip address 100.64.127.4/31
+   traffic-engineering bandwidth 100 mbps
    traffic-engineering administrative-group 4,7-100,testgrp
+   traffic-engineering srlg 16
+   traffic-engineering metric 2
+   traffic-engineering min-delay static 2 milliseconds
```

Comment:
This interface has no `traffic-engineering` command. Is that possible/valid? Is there a difference in EOS behavior having or not having that command? (If it is possible to not have it, I would expect it to be the "enabler" for the whole feature, and if so, we should keep the enabled flag in the doc table and set it properly depending on this.
We should also add a TODO for AVD 6.0.0 to set the `enabled` key as required in the schema.

Reply:
EOS accepts this, so we will keep the data model as is. The doc could still contain the enabled.